### DISCUSSION TITLE
Round battery `{power}`

### DISCUSSION THREAD

**Discussion by ok-nick** [Alexays/Waybar#1972] [Q&A]
How could I round the `{power}` format variable? Currently, the number is `21.392039904`, I just want `21`.

**Comment by MarkGhanz** [comment]
Same question here.

**Comment by Birdy2014** [comment]
You can format the variables using the [fmt Format String Syntax](https://fmt.dev/latest/syntax.html).
The syntax to round the power variable should be `{power:.0}`.

**Comment by FelixTraxler** [comment]
For me this worked: `{power:.0f}`. Also here is a working link to the fmt repo: https://github.com/fmtlib/fmt
